pebbles/static/js/controllers/UsersController.js

```diff
@@ -181,9 +197,35 @@ app.controller('UsersController', ['$q', '$scope', '$interval', '$uibModal', '$f
         }
     }]);
 
-app.controller('ModalQuotaController', function ($q, $scope, $modalInstance, Restangular, users) {
+app.controller('ModalQuotaController', function ($q, $scope, $modalInstance, Restangular, users, blueprints, groups) {
     $scope.users = users;
 
+    $scope.user_blueprint_info = function() {
+	    var user_groups = [];
+	    if(users.length != 0) {
+		$scope.blueprint_quota = users[0].blueprint_quota;
+		$scope.group_quota = users[0].group_quota;
+		user_groups = _.filter(groups, {'owner_eppn': users[0].eppn});
+	    }
+          
+          var user_blueprints = [];
+          var check_blueprints;
```

Comment:
If it is not initialized you can directly write var check_blueprints = _..... in line 215.